src/main/java/com/redhat/devtools/intellij/qute/psi/utils/PsiQuteProjectUtils.java

```diff
@@ -55,14 +52,31 @@ public static ProjectInfo getProjectInfo(Module javaProject) {
 		return new ProjectInfo(projectUri, projectDependencies
 				.stream()
 				.filter(projectDependency -> !javaProject.equals(projectDependency))
-				.map(projectDependency -> LSPIJUtils.getProjectUri(projectDependency))
+				.map(LSPIJUtils::getProjectUri)
 				.collect(Collectors.toList()), templateBaseDir);
 	}
 
 	private static String getTemplateBaseDir(Module javaProject) {
+		List<VirtualFile> resourcesDirs = ModuleRootManager.getInstance(javaProject).getSourceRoots(JavaResourceRootType.RESOURCE);
+		if (!resourcesDirs.isEmpty()) {
+			for (var dir : resourcesDirs) {
```

Comment:
I think we should remove TEMPLATES_BASE_DIR constants ad have:

 * TEMPLATES_FOLDER_NAME = "templates"
 * RESOURCES_BASE_DIR = "src/main"